OpenOversight/tests/conftest.py

```diff
@@ -48,6 +49,10 @@
     "Chief",
 ]
 
+DEPARTMENT_NAME = "Springfield Police Department"
+DEPARTMENT_TWO_NAME = "Chicago Police Department"
+DEPARTMENT_EMPTY_NAME = "Empty Police Department"
```

Comment:
🔬 I think `DEPARTMENT_NAME_XXXX` makes more sense as a naming motif.

`DEPARTMENT_NAME` -> `DEPARTMENT_NAME_SPRINGFIELD`
`DEPARTMENT_TWO_NAME` -> `DEPARTMENT_NAME_CHICAGO`
`DEPARTMENT_EMPTY_NAME` -> `DEPARTMENT_NAME_EMPTY`.

Reply:
Changed it slightly, hopefully make it a little more sense. But I want to make it clearer which department is considered the default one.